fluentd/configs.d/openshift/filter-k8s-events.conf

```diff
@@ -0,0 +1,24 @@
+<filter kubernetes.**> 
```

Comment:
@richm Do we need to consider moving this into ruby code for performance considerations?

Reply:
yes

Reply:
Maybe this is good enough for first pass with an enhancement to follow unless the move is trivial

Reply:
I can have the filter implemented in ruby today, its logic is not very complex, but I may need some help with the creation of the gem plugin and its productization. Or it can be implemented in the [ViaQ plugin](https://github.com/ViaQ/fluent-plugin-viaq_data_model). That would simplify localization but I am not sure if it is a good fit to implement this logic there.

Reply:
Since this is the implementation of the formatting for namespace in the viaq data model, I think this should be implemented in the viaq plugin.  Would really not like to create a new plugin/gem for this.  The alternative is to "drop in" the plugin code included with fluentd in /etc/fluent/plugins

Reply:
implement in viaq plugin https://github.com/ViaQ/fluent-plugin-viaq_data_model/pull/9#issuecomment-329044061